services/repository-managers/src/test/java/org/sagebionetworks/repo/manager/AuthorizationManagerImplTest.java

```diff
@@ -632,4 +631,19 @@ public void testCanSubscribeAnonymous() {
 		assertEquals(AuthorizationManagerUtil.accessDenied(ANONYMOUS_ACCESS_DENIED_REASON),
 				authorizationManager.canSubscribe(anonInfo, forumId, SubscriptionObjectType.FORUM));
 	}
+
+	@Test
+	public void testCanAccessMembershipInvitation() {
+		MembershipInvtnSubmission mis = new MembershipInvtnSubmission();
+		String misId = "1";
+		mis.setId(misId);
+		String teamId = "123";
+		mis.setTeamId(teamId);
+		mis.setInviteeId(userInfo.getId().toString());
+		mis.setMessage("Please join our team.");
+		MembershipInvtnSubmissionDAO mockMembershipInvtnSubmissionDAO = Mockito.mock(MembershipInvtnSubmissionDAO.class);
+		when(mockMembershipInvtnSubmissionDAO.get(misId)).thenReturn(mis);
+		ReflectionTestUtils.setField(authorizationManager, "membershipInvtnSubmissionDAO", mockMembershipInvtnSubmissionDAO);
+		assertTrue(authorizationManager.canAccess(userInfo, misId, ObjectType.MEMBERSHIP_INVITATION, ACCESS_TYPE.TEAM_MEMBERSHIP_UPDATE).getAuthorized());

```

Comment:
add  a comment above "// method under test"

Reply:
Tests to add:
- user is invitee, user is team mgr, user is neither
- various access types (READ, DELETE, other)